script/Ciel/Day 10/10_01_CIEL10_9.txt

```diff
@@ -561,7 +561,7 @@ Not even Ciel found out about this place, so how on earth did this woman know?
 [sha:f74d8fce2a19cc639c284721a754632e8f18f4dd]{
 -- Page 635, Offset 28878.
 -- だってあれ強者が弱者になっちゃう話じゃない？　愛が足りないんだってば、愛が」
-I mean, isn't it just the story someone strong becoming a total weakling?
+I mean, isn't the story just someone strong becoming a total weakling?
```

Comment:
Actually, I think I see the original intent now, was just missing a word. 

Isn't it just the story 'of' someone strong...